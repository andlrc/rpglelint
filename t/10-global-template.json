[
   {
      "column":8,
      "file":"t/10-global-template.rpgle",
      "line":7,
      "message":"global declaration 'not_tpl' is not allowed",
      "severity":"warning"
   }
]
