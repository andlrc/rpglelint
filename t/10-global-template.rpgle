// rpglelint: -Wglobal
dcl-ds tpl template;
  sub1 int(10);
  sub2 int(20);
end-ds;

dcl-ds not_tpl;
  sub1 int(10);
  sub2 int(20);
end-ds;
