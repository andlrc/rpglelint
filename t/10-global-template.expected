t/10-global-template.rpgle:7:8: warning: global declaration 'not_tpl' is not allowed [-Wglobal]
 dcl-ds not_tpl;
        ^
